include/z64player.h

```diff
@@ -158,7 +159,7 @@ typedef enum {
 } PlayerDoorType;
 
 
-#define PLAYER_LIMB_BUF_COUNT PLAYER_LIMB_MAX + 2 // 2 extra entries in limb buffers?
+#define LIMB_BUF_COUNT(limbCount) ((ALIGN16((limbCount) * sizeof(Vec3s)) + sizeof(Vec3s) - 1) / sizeof(Vec3s))
```

Comment:
It's pretty minor but I'd suggest keeping a `PLAYER_LIMB_BUF_COUNT` define to avoid duplication and keep things simpler in the struct

Reply:
so that is, just define `PLAYER_LIMB_BUF_COUNT` as LIMB_BUF_COUNT(PLAYER_LIMB_MAX)`?

Reply:
Yeah basically

Reply:
ill do that now, its a small change